lib/ts/recipe/thirdparty/utils.ts

```diff
@@ -55,3 +55,7 @@ function validateAndNormaliseSignInAndUpConfig(
         providers: config.providers,
     };
 }
+
+export function isFakeEmail(email: string): boolean {
+    return email.endsWith(".fakeemail.com");
```

Comment:
we need to change fakeemail domain to something like @stfakeemail.supertokens.com. So even the fake eamil generation process, we can use this domain.

Reply:
This change also needs to be done in other backend sdks

Reply:
We were generating it like this `${thirdPartyUserId}@${input.config.thirdPartyId}.fakeemail.com` would you need this updated? We will need to migrate the email ids if we choose to do so.

Reply:
yea, we should chanegs the domain to be something we control.

> We will need to migrate the email ids if we choose to do so.

Why would we need to migrate them? They would just be treated as regular emails.. would that be an issue? (Please think about this properly)

Reply:
I don't think migration would be required, but would be good to check the old format as well.

Reply:
as a side note, when discord users login with new backend, their email would change.